install.py

```diff
@@ -403,6 +428,16 @@ def parse_commandline_args():
 	parser.add_argument('--python', dest='python', default='python3', help=argparse.SUPPRESS)
 	# LXC Support
 	parser.add_argument('--container', dest='container', default=False, action='store_true', help='Use if you\'re creating inside LXC')
+
+	# for detecting dist; backward compatible if specify 2
+
+	parser.add_argument(
+		'--python-version',
+		dest='python_version',
+		default='3',
+		help='For detecting dist. Backward compatible if specify 2. Default is 3.'
+	)
```

Comment:
Instead of asking for user input, we can check the Python version used for executing the script by `sys.version_info.major`

Reply:
removed